dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard_progress_v2.feature

```diff
@@ -0,0 +1,32 @@
+@no_mobile
+Feature: Using the V2 teacher dashboard
+
+Scenario: Teacher can open and close Icon Key and details
+  Given I create an authorized teacher-associated student named "Sally"
+  And I complete the level on "http://studio.code.org/s/allthethings/lessons/2/levels/1"
+
+  When I sign in as "Teacher_Sally" and go home
+  And I get levelbuilder access
```

Comment:
Do we need levelbuilder access?

Reply:
Actually... kind of?  This was kinda a rabbit hole, but here is what I learned.  For most teachers, the `allthethings` course is not `assignable`.  However, if you have levelbuilder permissions, everything is assignable (see this [file](https://github.com/code-dot-org/code-dot-org/blob/6051a91aa58f48fd7d6e109fb1c6cc498d4903c6/dashboard/app/models/concerns/curriculum/assignable_course.rb#L4)).